Doc/library/email.message.rst

```diff
@@ -213,7 +213,7 @@ message objects.
          del msg['subject']
          msg['subject'] = 'Python roolz!'
 
-      If the :mod:`policy` defines certain headers to be unique (as the standard
+      If the :mod:`policy <email.policy>` defines certain headers to be unique (as the standard
```

Comment:
```suggestion
      If the :mod:`~email.policy` defines certain headers to be unique (as the standard
```

Reply:
In this case "policy" is not a shortened module name, but just a term. In translated documentation it will be translated.

Reply:
Ahh! Good point, I hadn't thought about translations. The current text should stand.